spec/functional/resource/group_spec.rb

```diff
@@ -361,7 +361,7 @@ def create_group
   end
 
   describe "group modify action", :not_supported_on_solaris do
-    let(:spec_members){ ["Gordon", "Eric", "Anthony"] }
+    let(:spec_members){ ["gordon", "eric", "anthony"] }
```

Comment:
actually these are now probably terrible names since we really create and delete them, and we'll hose someone's accounts...